scripts/index.js

```diff
@@ -41,3 +41,366 @@ navListItem.forEach((listItem) => {
+  const newPar = document.createElement('p');
+  newPar.classList.add('popup-description');
+
+  const popBtnCntnr = document.createElement('div');
+  popBtnCntnr.classList.add('pop-btn-contnr');
+  const btn1 = document.createElement('a');
+  btn1.classList.add('popup-button', 'live-button');
+  btn1.textContent = 'See Live';
+  btn1.target = '_blank';
+  const btn2 = document.createElement('a');
+  btn2.classList.add('popup-button', 'src-button');
+  btn2.target = '_blank';
+  btn2.textContent = 'See Source';
+  popBtnCntnr.appendChild(btn1);
+  popBtnCntnr.appendChild(btn2);
+
+  popup.appendChild(closeIcon);
+  popup.appendChild(header);
+  popup.appendChild(techContainer);
+  popup.appendChild(newDiv);
+  popup.appendChild(thumbnailScroll);
+  popup.appendChild(newPar);
+  popup.appendChild(popBtnCntnr);
+
+  return popup;
+}
+
+// function to create popup for mobile version
+function createDesktopPopup() {
+  const desktopModal = document.createElement('div');
+  desktopModal.classList.add('desktop-modal');
+  const desktopPop = createMobilePopup();
+  const closeIcon = desktopPop.querySelector('.popup-close-btn');
+  closeIcon.onclick = function () {
+    const container = document.querySelector('.projects-container');
+    const pop = document.querySelector('.desktop-modal');
+    container.removeChild(pop);
+    document.body.style.overflow = 'auto';
+  };
+  desktopPop.classList.remove('popup-container');
+  desktopPop.classList.add('popup-desktop');
+  const dtheader = desktopPop.querySelector('.popup-header');
+  dtheader.classList.add('dt-pop-header');
+  const dtdescr = desktopPop.querySelector('.popup-description');
+  dtdescr.classList.add('dt-pop-descr');
+  const btnCntnr = desktopPop.querySelector('.pop-btn-contnr');
+  btnCntnr.classList.add('dt-btn-cntnr');
+  const dtliveBtn = desktopPop.querySelector('.live-button');
+  dtliveBtn.classList.add('dt-live-button');
+  const dtSrcBtn = desktopPop.querySelector('.src-button');
+  dtSrcBtn.classList.add('dt-src-button');
+
+  const projNav = document.createElement('div');
+  projNav.classList.add('proj-nav');
+  const leftProjBtn = document.createElement('button');
+  leftProjBtn.classList.add('proj-nav-left');
+  leftProjBtn.textContent = 'Prevoius project';
+  const rightProjBtn = document.createElement('button');
+  rightProjBtn.classList.add('proj-nav-right');
+  rightProjBtn.textContent = 'Next project';
+  projNav.append(leftProjBtn);
+  projNav.append(rightProjBtn);
+  desktopPop.append(projNav);
+  desktopModal.appendChild(desktopPop);
+  return desktopModal;
+}
+
+function createPopup(project) {
+  let popupwindow;
+  if (window.innerWidth <= 768) {
+    popupwindow = createMobilePopup();
+  }
+  if (window.innerWidth > 768) {
+    popupwindow = createDesktopPopup();
+  }
+  const container = document.querySelector('.projects-container');
+  container.appendChild(popupwindow);
+  loadNewProj(project);
+  document.body.style.overflow = 'hidden';
+}
```

Comment:
As requested by the previous reviewer, kindly ensure that your design matches that of the figma design. The skills  boxes containing the `Ruby on rails`, `HTML/CSS`, and `JavaScript` looks bigger than the actual design. :confused: This is because the desktop and the mobile modals are being created separately hence it is not working responsively and the media queries from the stylesheet is not working. If you open the modal popup from a desktop screen width it maintains the same shape and dimensions without switching to the mobile popup modal for all screens of both desktop and mobile and vice versa. :wink: 
Kindly ensure that you create only one modal popup and use css media queries to switch from mobile to desktop and vice versa instead of creating two popup windows. :+1: 

### Mobile version when created from lines 304-306 (index.js)

![Screenshot from 2022-12-21 19-01-59](https://user-images.githubusercontent.com/65233860/208986014-f834ac66-ba72-4ed4-9ac6-0194990374cf.png)

### Actual design 

![Screenshot from 2022-12-21 19-25-48](https://user-images.githubusercontent.com/65233860/208986985-7d3a3b13-cae1-4150-8a05-8c10c5ffe827.png)




.....................................................................................................................................................

## Please kindly check these other images for optional perspective

### Mobile version when created from lines 304-306 (index.js)

![Screenshot from 2022-12-21 19-02-21](https://user-images.githubusercontent.com/65233860/208986034-3b0c11d1-a8d1-41f0-8561-b8e65b85a749.png)


### Desktop version If created from lines 300-303 (index.js)

![Screenshot from 2022-12-21 19-02-58](https://user-images.githubusercontent.com/65233860/208986058-8c066fd1-1f9f-4731-8eb3-60850873ea97.png)